tests/integration/store/rootmulti/rollback_test.go

```diff
@@ -63,7 +64,7 @@ func TestRollback(t *testing.T) {
 		app.Commit()
 	}
 
-	require.Equal(t, ver0+10, app.LastBlockHeight())
+	assert.DeepEqual(t, ver0+10, app.LastBlockHeight())
```

Comment:
Do we need to use deepequal for normal value checks? I believe equal should suffice.

Reply:
Same comment for the others PRs actually 🙈

Reply:
Can you check in your other PRs too 👀